data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,319 @@ mission "Wanderers: Pre-Colony"
 			`	As you land on the outskirts of this settlement, one of the workers comes up to you as you leave your ship. "[Greetings, welcome], it is [delightful, pleasing] to see you have come to show your interest in our exploration. Unfortunately, there exists much [hazard, risk] on a world we know so little about. This will be our staging ground to [explore, catalog] this new region, but it will take some time to be assured of the [safety, security] of our outpost. I suggest you take time to [orient, locate] yourself in these systems while we set up before returning to help. Of course you are welcome when we have [determined, established] safety, there will be many ships needed to help heal the many ruined planets here.`
 				decline
 
+
+mission "Wanderers: Spera Anatrusk Outfitter 1"
+	landing
+	name "Establish outfitter on Spera Anatrusk"
+	description "Return to the Wanderer capital on <destination>."
+	source "Spera Anatrusk"
+	destination "Vara K'chrai"
+
+	to offer
+		has "event: wanderers: the eye opens"
+
+	on offer
+		log `The Wanderers are establishing a colony on <planet> in Korath space, but they need help getting the most basic infastructure established.`
+		conversation
+			`The landing pads at the settlement are packed earth and gravel, mostly created by the ships themselves. There is a large tent for the workers to meet and rest with a battered Strong Wind acting as a command center. You can see a small crew of Wanderers hammering forms together with a concrete mixer running nearby. It's a far cry from the clean and elegant spaceports the Wanderers are used to, but not far off some of the Dirt Belt and anarchist worlds you've known.`
+			`	Something of an argument is taking place in front of a tent at the edge of landing area. There are a lot of wings flapping and raised voices, but they aren't speaking Hai, so you have no idea what it's about.`
+			choice
+				`	(Go over and investigate.)`
+					goto investigate
+				`	(Leave and explore this part of space.)`
+					decline
```

Comment:
I'm no so sure about these missions being required to get an outfitter working if you can just decline them and then have the outfitter never be built. Even if you're able to decline these missions, the outfitters should eventually be finished anyway.

Reply:
Have added checks for decline, fail and abort to insure that colony event is triggered in any case.